PWGHF/Tasks/CMakeLists.txt

```diff
@@ -29,12 +29,12 @@ o2physics_add_dpl_workflow(task-mc-validation
                     PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore
                     COMPONENT_NAME Analysis)
 
-o2physics_add_dpl_workflow(task-sel-optimisation
-                    SOURCES taskSelOptimisation.cxx
-                    PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore
-                    COMPONENT_NAME Analysis)
-
```

Comment:
Why?

Reply:
The task is obsolete and not maintained anymore (I was the developer of it). I did not want to delete it from O2Physics since if somebody wants to update it to use it can be done, but in the meantime it makes no sense to keep it in the compilation, since it is outdated.

Reply:
OK, but let's just comment it out then, so that it's easy to add back in when needed.

Reply:
Done! It was also very easy to re-include it without commenting it anyway ;)

Reply:
Thanks